miden-lib/asm/note_scripts/SWAP.masm

```diff
@@ -5,6 +5,8 @@ use.miden::contracts::wallets::basic->wallet
 # =================================================================================================
 
 const.PRIVATE_NOTE=2
+# The hint for "Always" is 1, so the number for the hint is 1 << 32 (32 bits for a blank payload)
+const.EXECUTION_HINT_ALWAYS=4294967296
```

Comment:
Looking at this, I'm wondering if we should change the encoding scheme of the hint a bit so that `ALWAYS` maps to just `1` (we could do this by putting the tag into the least significant bits).

Reply:
Changed the scheme to place the tag on the least significant bits